app/assets/stylesheets/scaffolds.scss

```diff
@@ -93,3 +93,16 @@ body.dark-theme {
 body.trusted-status-true .trusted-visible-block {
   display: flex !important;
 }
+
+// The .admin-help-button class is included in this file since it
+// is referenced in both the admin and application layout.
+
+.admin-help-button {
+  position: fixed;
+  margin-bottom: env(safe-area-inset-bottom);
+  border-radius: 50px;
```

Comment:
I'm not sure what these would be as a utility class. Do we have any documentation outside of the Storybook documentation that could help me figure this out?

Reply:
@Ridhwana you're very right. we lack proper documentation for both: utility classes and components in general.. i work on that time to time but it's just hard to fully commit to writing documentation - I guess we're spread thin.

They way you solve, even though it's pretty painful and not efficient, is actually quite sensible with current state of things. I'd only add that maybe instead of point 3 (or as an addition..?) I'd go to [_generator.scss](https://github.com/forem/forem/blob/main/app/assets/stylesheets/config/_generator.scss) file (which is responsible for generating all our utility-first classes) and hit CMD+F to search for specific CSS property.

---

To be honest, I don't know what's the plan for [official Tailwind adoption](https://github.com/forem/rfcs/pull/51) anymore... I've been trying to push it forward for very long time but it's been a bit frustrating so [I gave up](https://github.com/forem/rfcs/pull/51#issuecomment-954709396)... Maybe we will revisit this some day 🤷 .